std_msgs/Header header
string data_e
string finish_c
string video_name_p
sensor_msgs/Image image_e
sensor_msgs/Image image_p
float32 target_pos_x
float32 target_pos_y

